.circleci/config.yml

```diff
@@ -226,6 +231,12 @@ workflows:
           requires:
             - test
 
+  deploy-docs:
+    jobs:
+      - deploy:
+          filters:
+            <<: *filters_master_branch
```

Comment:
I suspect this will deploy a blank directory as the workflow is incomplete. My understanding is that this won't inherit the existing `build-test` workflow used for all branch builds.

This deploy step could be added to the existing `build-test` workflow with the addition of the master branch filter.

Reply:
You are right, tested in a dummy repo 👍